style.css

```diff
@@ -0,0 +1,228 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
- [ ] Oops!!,  I think your last modifications have broken your app, please review it to fix the broken parts
<img width="378" alt="1" src="https://user-images.githubusercontent.com/77843050/228937170-f920f882-b35a-430a-b081-876ea2820bf3.png">